src/main/java/org/hisp/dhis/utils/Randomizer.java

```diff
@@ -0,0 +1,199 @@
+package org.hisp.dhis.utils;
+
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Random;
+
+public interface Randomizer {
+    /**
+     * Extracts a random element from a list
+     *
+     * @param list a List
+     * @param <T>  type
+     * @return a random element from the list
+     */
+    default <T> T randomElementFromList( List<T> list )
+    {
+        return list.get( randomInt( list.size() ) );
+    }
+
+    /**
+     * Return the underlying {@link Random } object
+     *
+     * @return a {@link Random } object
+     */
+    Random getRandom();
+
+    /**
+     * Generates a random integer
+     *
+     * @param bound the maximum boundary for the generated int
+     * @return a random int
+     */
+    public int randomInt(int bound);
+
+    /**
+     * Extracts a random sublist of n elements from a list
+     *
+     * @param list a List
+     * @param size the size of the sublist
+     * @param <T>  type
+     * @return a random sublist
+     */
+    <T> List<T> randomElementsFromList(List<T> list, int size);
+
+    /**
+     * Returns random string containing 6 alphabetical characters.
```

Comment:
does this mean the string is always 6 characters long or just that it has at least 6 letters?